web/text_layer_builder.js

```diff
@@ -168,11 +158,18 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
         left = tx[4] + (fontAscent * Math.sin(angle));
         top = tx[5] - (fontAscent * Math.cos(angle));
       }
+      // Save info about the div in the geom for fast access.
+      geom.div_left = left;
```

Comment:
Nit: please don't use underscore in variable names, they are reserved for constants, see https://github.com/mozilla/pdf.js/wiki/Style-Guide#naming.

This also applies in multiple other places in this patch.